### PR TITLE
Feat add slurm

### PR DESCRIPTION
**Pull request by tdivoll**
This PR add a global.cylc config file and adds slurm directives for each task. There is also a new task to copy over the global config from the repo to a location that Cylc is looking for. 

I tested a few directive configs and set the defaults to seemingly reasonable numbers of GB and cores for each task, but we could do more extensive testing in a new PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,34 +10,29 @@ The [Satellite Overpass Identification Tool](https://zenodo.org/record/6475619#.
 
 ## Cylc to run the pipeline
 
-Cylc is used to encode the entire pipeline from start to finish and relies on the command line scripts to automate the workflow. The `config/cylc_hpc/flow.cylc` file should be suitable for runs on HPC systems. To run Cylc locally, the `config/cylc_local_flow.cylc` file is used.
+Cylc is used to encode the entire pipeline from start to finish and relies on the command line scripts to automate the workflow. The `config/cylc_hpc/flow.cylc` file should be suitable for runs on HPC systems. The default pipeline is built to run on Brown's Oscar HPC and each task is submitted as it's own batch job. To run Cylc locally, the `config/cylc_local_flow.cylc` file is used.
```
Comment (cpaniaguam) [inline README.md:13]:
```suggestion
Cylc is used to encode the entire pipeline from start to finish and relies on the command line scripts to automate the workflow. The `config/cylc_hpc/flow.cylc` file should be suitable for runs on HPC systems. The default pipeline is built to run on Brown's Oscar HPC and each task is submitted as its own batch job. To run Cylc locally, the `config/cylc_local_flow.cylc` file is used.
```

File: README.md [hunk 2]
```diff
@@ -10,34 +10,29 @@ The [Satellite Overpass Identification Tool](https://zenodo.org/record/6475619#.
...
 
 ### Running the Cylc pipeline on Oscar
 
 1. - [ ] Connect to Oscar from VS Code
     * [use this guide](https://docs.ccv.brown.edu/oscar/connecting-to-oscar/remote-ide)
 
-2. Move to a compute node
-   - [ ] `interact -n 20 -t 24:00:00 -m 32g`
-    * this will start a compute session for 1 day with 32 GB memory and 20 cores
-    * see [here](https://docs.ccv.brown.edu/oscar/submitting-jobs/interact) for more options
```
Comment (cpaniaguam) [inline README.md:23]: So Cylc can manage this kind of thing on its own?

File: config/cylc_hpc/flow.cylc [hunk 3]
```diff
@@ -42,40 +49,99 @@
...
+    
     [[exportH5]]
         script = """
             singularity exec --bind $preprocess_dir:/tmp,$report_dir:/usr/local/bin/../report $project_dir/icefloetracker-julia.simg $julia_exec -t auto /usr/local/bin/ice-floe-tracker.jl makeh5files --pathtosampleimg $fetchdata_dir/truecolor/$(ls $fetchdata_dir/truecolor | head -1) --resdir /tmp
         """
+        platform = oscar
+        execution time limit = PT1H
+        [[[directives]]]
+            --mem = 32G
+            --cpus-per-task = 24
```
Comment (cpaniaguam) [inline config/cylc_hpc/flow.cylc:146]: So this kind of thing cannot be set globally?

**Review by cpaniaguam — APPROVED** [review]
Just some clarifying questions and typo.

File: README.md (hunk 2 above)
Comment (tdivoll) [reply README.md:23]: It is covered by a combination of VS Code connecting to a compute node and then each task being submitted to slurm with memory and cores.

File: config/cylc_hpc/flow.cylc (hunk 3 above)
Comment (tdivoll) [reply config/cylc_hpc/flow.cylc:146]: I will add some defaults to the global.cylc file and then we can also specify directives in each block to override the defaults.

File: config/cylc_hpc/flow.cylc [hunk 4]
```diff
@@ -42,40 +49,99 @@
             mkdir -p $tracker_dir
             mkdir -p $h5_dir
         """
+        platform = oscar
+        execution time limit = PT1H
+        [[[directives]]]
+            --mem = 4G
+            --cpus-per-task = 6
```
Comment (hollandjg) [inline config/cylc_hpc/flow.cylc:56]: You probably don't want to have more than one CPU for this – you're just making directories. You might not even need to dispatch via slurm – the scheduler should be able to handle this task.

File: README.md [hunk 5]
```diff
@@ -127,6 +122,7 @@ If you need to change parameters and re-run a workflow, first do:
     cylc clean <workflow-name>
     ```
    - [ ] Then, proceed to install, play, and open the TUI
+   - [ ] To rerun in one line: ```cylc stop --now <workflow-name> && cylc clean <workflow-name> && cylc install -n <workflow-name> ./config/cylc_hpc && cylc play <workflow-name> && cylc tui <workflow-name>```
```
Comment (hollandjg) [inline README.md:125]: you might want to include `cylc validate` in here – that checks your flow file before you start

**Review by hollandjg — COMMENTED** [review]
Looks good! just a couple of comments.

File: config/cylc_hpc/flow.cylc (hunk 3 above)
Comment (tdivoll) [reply config/cylc_hpc/flow.cylc:146]: good point, but it's such a tiny task that maybe it's okay?

**Pull request merged**